Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/JPath/Evaluator.cs

```diff
@@ -656,7 +669,7 @@ public bool Matches(string lhsStr, Result rhs)
             var rhsVal = ToScalar(rhs);
             switch (rhsVal)
             {
-                case string str: return lhsStr.Contains(str);
+                case string str: return lhsStr.ToUpperInvariant().Contains(str.ToUpperInvariant());

```

Comment:
>Contains [](start = 66, length = 8)

why is it `contains` and not `==` (given the method name is `matches`)

Reply:
Either semantics would make sense.  I originally chose 'contains' because I though that would be more convenient in practice